x-pack/plugins/lists/public/exceptions/components/builder/entry_renderer.tsx

```diff
@@ -268,6 +273,25 @@ export const BuilderEntryItem: React.FC<EntryItemProps> = ({
     }
   };
 
+  // show this when wildcard filename with matches operator
+  const getWildcardWarning = (precedingWarning: string): React.ReactNode => {
+    return (
+      <p>
+        {precedingWarning}{' '}
+        <EuiIconTip
+          type="iInCircle"
+          content={
+            <FormattedMessage
+              id="xpack.lists.exceptions.builder.exceptionMatchesOperator.warningMessage.wildcardInFilepath"
```

Comment:
@ashokaditya thinking about this message more - how difficult is it add a conditional to our `process.name` and `file.name` additions if the user has already added the field themselves as a result of this message?

My fear is that we try to help them, but create another invalid rule if they add a `file.name` themselves and we also add one as a duplicate.

Talking about this pr: https://github.com/elastic/kibana/pull/127432

Reply:
Good point. I'll see if I can add a conditional for the two fields in the warning.